ci/scripts/run-e2e-test.sh

```diff
@@ -115,7 +114,6 @@ pkill python3
 
 sqllogictest -p 4566 -d dev './e2e_test/udf/alter_function.slt'
 sqllogictest -p 4566 -d dev './e2e_test/udf/graceful_shutdown_python.slt'
-sqllogictest -p 4566 -d dev './e2e_test/udf/always_retry_python.slt'
```

Comment:
There's one concern: We cannot distinguish which tests are run in which pipeline. Ideally we should separate them into different directories like `e2e_test_slow`.

Reply:
Sure, I think we could do that.

Reply:
Done

Reply:
09b94e50de22674da69cf47bed0ec8c4ad3b5bd1